app/services/notifications/new_comment/send.rb

```diff
@@ -49,7 +49,10 @@ def call
             user_ids: targets,
             title: "@#{comment.user.username}",
             body: "Re: #{comment.parent_or_root_article.title.strip}",
-            payload: { url: URL.url(url_path) },
+            payload: {
+              url: URL.url(url_path),
+              type: "Notifications::NewComment"
```

Comment:
should we use a name that's independent from the name of the Ruby class? In case the class gets... renamed.
We might also want to test this part, regardless of the string name

Reply:
Good point @rhymes: I'll make these types more human readable.